modules/images/src/load-image/load-image.js

```diff
@@ -0,0 +1,21 @@
+import {isBrowser, loadImage as loadImageBrowser} from '@loaders.gl/core';
+import getPixels from 'get-pixels';
+import util from 'util';
+
+export function loadImage(url, options = {}) {
+  if (isBrowser) {
+    return loadImageBrowser(url, options);
```

Comment:
This only supports main thread (eventually calls `loadToHTMLImage`). Is it intended?